conductr_cli/conduct.py

```diff
@@ -109,24 +110,26 @@ def build_parser():
     # Sub-parser for `events` sub-command
     events_parser = subparsers.add_parser('events',
                                           help='show bundle events')
-    events_parser.add_argument('--service',
-                               default='http://{}:9210'.format(default_ip),
-                               help='Events service address')
+    add_ip_and_port(events_parser)
+    events_parser.add_argument('--count',
+                               type=int,
+                               default=10,
+                               help='The number of events to fetch, defaults to 10, maximum is 100')
     events_parser.add_argument('bundle',
-                               help='The ID of the bundle')
-    # events_parser.set_defaults(func=conduct_events.events) FIXME
-    events_parser.set_defaults(func=lambda x: print('This command is not yet available. Consult ConductR logs.'))
+                               help='The ID or name of the bundle')
+    events_parser.set_defaults(func=conduct_events.events)
 
     # Sub-parser for `logs` sub-command
     logs_parser = subparsers.add_parser('logs',
                                         help='show bundle logs')
-    logs_parser.add_argument('--service',
-                             default='http://{}:9210'.format(default_ip),
-                             help='Logs service address')
+    add_ip_and_port(logs_parser)
+    logs_parser.add_argument('--count',
```

Comment:
@huntc - I have introduced the `--count` switch to indicate how many events/logs to be fetched. Please let me know if you think this is ok.